### DISCUSSION TITLE
Adding zee editor to nixpkgs

### DISCUSSION THREAD

**Discussion by booklearner** [zee-editor/zee#89] [Show and tell]
Hey! For anyone else using `nix`, I've created a PR to add the zee editor to the package repositories.

https://github.com/NixOS/nixpkgs/pull/185449

**Comment by kevinmatthes** [comment]
Thank you, @booklearner!  I just opened an issue about a new release such that the progress since version 0.3.2 can be made accessible for everyone from the official package registries.

**Reply by booklearner** [reply]
Nice (reference: https://github.com/zee-editor/zee/issues/92). Feel free to ping me when there's some traction and I'd be happy to keep the nix package up to date!
